atoms/Cargo.toml

```diff
@@ -1,10 +1,12 @@
 [package]
 name = "servo_atoms"
```

Comment:
Maybe this should be called `stylo_atoms`?

Reply:
Yeah, would be happy to go with that name.